csrc/preprocess/transform/pad.cpp

```diff
@@ -57,17 +63,14 @@ Result<Value> PadImpl::Process(const Value& input) {
       std::array padding{0, 0, pad_w - width, pad_h - height};
 
       OUTCOME_TRY(output_tensor, PadImage(tensor, padding));
-
       output["pad_size_divisor"] = arg_.size_divisor;
       output["pad_fixed_size"].push_back(pad_h);
       output["pad_fixed_size"].push_back(pad_w);
     } else {
-      std::array padding{0, 0, arg_.size[1] - width, arg_.size[0] - height};
-
+      std::array padding{0, 0, max(0, arg_.size[1] - width), max(0, arg_.size[0] - height)};
```

Comment:
I am not sure if this is necessary to add max here. A valid config will resize the image before padding, this should never happened.